reactor-netty-core/src/main/java/reactor/netty/Metrics.java

```diff
@@ -142,11 +142,21 @@
 	 */
 	public static final String TOTAL_CONNECTIONS = ".total.connections";
```

Comment:
these are now unused, right?

Reply:
These are used by the `ConnectionProvider` and cannot be changes as this is a backwards incompatible change.